charts/fluent-operator/values.yaml

```diff
@@ -95,7 +95,7 @@ fluentd:
   output:
     es:
       enable: false
-      host: elasticsearch-logging-data.kubesphere-logging-system.svc
+      host: elasticsearch-logging-data.fluent.svc
```

Comment:
we can keep the es namespace as it is before : elasticsearch-logging-data.kubesphere-logging-system.svc